src/changelog.ts

```diff
@@ -130,7 +128,7 @@ export default class Changelog {
     for (const commit of commits) {
       const issue = commit.githubIssue;
       const login = issue && issue.user && issue.user.login;
-      // If a list of `ignoreCommitters` is provided in the lerna.json config
+      // If a list of `ignoreCommitters` is provided in the lernaon config
```

Comment:
I don't know why this line changed, but if you're changing it then it should probably be correct. Do we have a bit in the readme that describes this?

Reply:
Woot, i revert this